sequencing-server/src/app.ts

```diff
@@ -123,17 +123,18 @@ app.post('/put-dictionary', async (req, res, next) => {
   logger.info(`command-lib generated - path: ${commandDictionaryPath}`);
 
   const sqlExpression = `
-    insert into command_dictionary (command_types_typescript_path, mission, version)
-    values ($1, $2, $3)
+    insert into command_dictionary (command_types_typescript_path, mission, version, parsed_json)
```

Comment:
I know that @Mythicaeda is going to drop the `update_at` column, but I think you guys are working that out.

Reply:
Yeah, it's part of another PR!